spec/fixtures/form1010_ezr/valid_form.json

```diff
@@ -0,0 +1,119 @@
+  "veteranAddress": {
+    "street": "7321 SW 7th St",
+    "street2": "",
+    "street3": "",
+    "city": "Ontario",
+    "country": "CAN",
+    "state": "ON",
+    "provinceCode": "ProvinceName",
+    "postalCode": "21534"
+  },
+  "veteranHomeAddress": {
+    "street": "123 NW 5th St",
+    "street2": "",
+    "street3": "",
+    "city": "Ontario",
+    "country": "CAN",
+    "state": "ON",
+    "provinceCode": "ProvinceName",
+    "postalCode": "21231"
+  },
+  "homePhone": "1231241234",
+  "mobilePhone": "1235551234",
+  "email": "foo@example.com",
+  "maritalStatus": "Married",
+  "providers": [
+    {
+      "insuranceName": "MyInsurance",
+      "insurancePolicyHolderName": "FirstName ZZTEST",
+      "insurancePolicyNumber": "P1234",
+      "insuranceGroupCode": "G1234"
+    }
+  ],
+  "isMedicaidEligible": true,
+  "isEnrolledMedicarePartA": true,
+  "medicarePartAEffectiveDate": "1999-10-16",
+  "medicareClaimNumber": "873462432",
+  "spouseFullName": {
+    "first": "FirstSpouse",
+    "middle": "MiddleSpouse",
+    "last": "LastSpouse",
+    "suffix": "Sr."
+  },
+  "spouseSocialSecurityNumber": "111221234",
+  "spouseDateOfBirth": "1990-04-06",
+  "dateOfMarriage": "2003-05-10",
+  "cohabitedLastYear": false,
+  "provideSupportLastYear": false,
+  "spouseAddress": {
+    "street": "123 NW 8th St",
+    "street2": "",
+    "street3": "",
+    "city": "Dulles",
+    "country": "USA",
+    "state": "VA",
+    "postalCode": "20101-0101"
+  },
+  "dependents": [
+    {
+      "fullName": {
+        "first": "FirstChildA",
+        "middle": "MiddleChildA",
+        "last": "LastChildA",
+        "suffix": "Jr."
+      },
+      "dependentRelation": "Stepson",
+      "socialSecurityNumber": "111229876",
+      "becameDependent": "1992-04-07",
+      "dateOfBirth": "1982-05-05",
+      "disabledBefore18": true,
+      "attendedSchoolLastYear": true,
+      "dependentEducationExpenses": 45.2,
+      "grossIncome": 991.9,
+      "netIncome": 981.2,
+      "otherIncome": 91.9
+    },
+    {
+      "fullName": {
+        "first": "FirstChildB",
+        "middle": "MiddleChildB",
+        "last": "LastChildB",
+        "suffix": "Sr."
+      },
+      "dependentRelation": "Stepdaughter",
+      "socialSecurityNumber": "222111234",
+      "becameDependent": "2003-04-07",
+      "dateOfBirth": "1996-03-07",
+      "disabledBefore18": false,
+      "attendedSchoolLastYear": true,
+      "dependentEducationExpenses": 1198.11,
+      "grossIncome": 791.9,
+      "netIncome": 781.2,
+      "otherIncome": 71.9
+    }
+  ],
+  "veteranGrossIncome": 123.33,
+  "veteranNetIncome": 90.11,
+  "veteranOtherIncome": 10.1,
+  "spouseGrossIncome": 64.1,
+  "spouseNetIncome": 35.1,
+  "spouseOtherIncome": 12.3,
+  "deductibleMedicalExpenses": 33.3,
+  "deductibleFuneralExpenses": 44.44,
+  "deductibleEducationExpenses": 77.77,
+  "vaMedicalFacility": "608",
+  "isSpanishHispanicLatino": true,
+  "isEssentialAcaCoverage": false,
```

Comment:
these 2 fields shouldn't be in the form, because the frontend is not going to send them